dashboard/src/pages/TreeDetails/TreeDetailsFilterList.tsx

```diff
@@ -32,7 +32,8 @@ const TreeDetailsFilterList = ({
 
   const onClickItem = useCallback(
     (flatValue: string, _: number) => {
-      const [field, value] = flatValue.split(':');
+      const [field, ...rest] = flatValue.split(':');
+      const value = rest.join(':');
```

Comment:
`split` has another argument called limit which can make split less greedy, so you don't need to rejoin again

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split

Reply:
wow, nice tip!!!!

Reply:
If I use `limit`, I still get the same error because your functionality is to "stops when limit entries have been placed in the array". So the return of `"configs:cros://chromeos-6.6/x86_64/chromeos-intel-pineview.flavour.config
".split(":", 2)` would be `["configs", "cros"]`